EIPS/eip-1.md

```diff
@@ -119,6 +119,8 @@ Each EIP must begin with an [RFC 822](https://www.ietf.org/rfc/rfc822.txt) style
 
 ` title:` *EIP title*
 
+` description:` *Simple summary of the EIP*
```

Comment:
Consider adding this text (from the template):
> Provide a simplified and layman-accessible explanation of the EIP. Imagine an email subject line, GitHub PR title, or forum post title.

Reply:
Done.